components/SelectItemInput.tsx

```diff
@@ -0,0 +1,71 @@
+import { ChevronDownIcon } from '@chakra-ui/icons';
+import {
+  Button,
+  HStack,
+  Image,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+  Text,
+  Tooltip,
+} from '@chakra-ui/react';
+
+import { Item } from '@/utils/types';
+
+type SelectItemInputProps = {
+  selectedItem: Item | null;
+  setSelectedItem: (item: Item) => void;
+  items: Item[];
+};
+
+export const SelectItemInput: React.FC<SelectItemInputProps> = ({
+  selectedItem,
+  setSelectedItem,
+  items,
+}) => {
+  return (
+    <Menu>
+      <Tooltip label={items.length === 0 ? 'No items available' : ''}>
```

Comment:
Wonder if we should have this say, "No transferrable items available" instead, so it's a bit more specific

Reply:
fixed in [846ae47](https://github.com/raid-guild/character-sheets-frontend/pull/170/commits/846ae471e0454be0c7b863960a53c5166866b938)